src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -303,6 +304,66 @@ class Browser extends DashboardView {
     }
   }
 
+  saveNewRow(){
+    const obj = this.state.newObject;
+    if (!obj) {
+      return;
+    }
+
+    obj.save(null, { useMasterKey: true }).then(
+      objectSaved => {
+        let msg = objectSaved.className + ' with id \'' + objectSaved.id + '\' created';
+        this.showNote(msg, false);
+
+        const state = { data: this.state.data };
+        const relation = this.state.relation;
+        if (relation) {
+          const parent = relation.parent;
+          const parentRelation = parent.relation(relation.key);
+          parentRelation.add(obj);
+          const targetClassName = relation.targetClassName;
+          parent.save(null, { useMasterKey: true }).then(
+            () => {
+              this.setState({
+                newObject: null,
+                data: [obj, ...this.state.data],
+                relationCount: this.state.relationCount + 1,
+                counts: {
+                  ...this.state.counts,
+                  [targetClassName]: this.state.counts[targetClassName] + 1
+                }
+              });
+            },
+            error => {
+              let msg = typeof error === "string" ? error : error.message;
+              if (msg) {
+                msg = msg[0].toUpperCase() + msg.substr(1);
+              }
+              obj.set(attr, prev);
```

Comment:
Hey @sadakchap I've just run into this line while working on #2003 . I noticed that `attr` and `prev` are not defined so it's giving me an error. 

![image](https://user-images.githubusercontent.com/8172495/149674479-939a30d6-d21e-4997-a011-e3d1a1231edb.png)

I'm hesitant to change this line to `obj.revert()` but I'd like to check first with you before proceeding.

Reply:
Hey @RaschidJFR, I think we can safely remove this line. Earlier we use to save new row(making an api call) everytime a user was updating any cell. Incase of error, this line would revert current selected cell value. But, now I think we don't need it as this function gets call when user hits save btn.